hack/run-e2e.sh

```diff
@@ -214,9 +214,10 @@ function e2e::image_load() {
         $E2E_IMAGE
     )
     if [ "$PROVIDER" == "kind" ]; then
+        local nodes=$($KIND_BIN get nodes --name $CLUSTER | grep -v 'control-plane$')
         echo "info: load images ${images[@]}"
         for n in ${images[@]}; do
-            $KIND_BIN load docker-image --name $CLUSTER $n
+            $KIND_BIN load docker-image --name $CLUSTER $n --nodes $(hack::join ',' ${nodes[@]})
```

Comment:
this is a time-consuming process, we can skip master node to speed it up a little